src/App.js

```diff
@@ -30,29 +31,40 @@ function App() {
     [155, 156, 157],
     [158, 159, 160]
   ];
-
-  // function to compare users score to dealers
-  // function to test if users score is less than / equal 21
   
+  // object holding the dealer's pokemon
   const dealerPokemon = { 
     name: "mr-mime", 
     sprite: "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/122.png"
   };
+
+  const handleGameStart = () => {
+    setGameState(true);
+  }
   
   useEffect(() => {
-    // generate random pokemon index from availablePokemon array, make the API call, and set the playerPokemon state
-    const getPokemon = async (id) => {
-      const pokemonId = await fetch(`https://pokeapi.co/api/v2/pokemon/${id}/`);
-      const pokemonObject = await pokemonId.json();
+    // generate random pokemon family from availablePokemon array 
+    const pokemonFamily = randomizer(availablePokemon);
+
+    // make the API calls for all three pokemon in the evolution line
+    const chosenFamily = pokemonFamily.map(async (pokemonId) => {
```

Comment:
This could be "theoretically" faster if we use Promise.all, (because we're waiting for each call to finish) but I guess the difference would not be worth re-writing